examples/build_graph/simple_kg_builder_from_pdf_ollama.py

```diff
@@ -0,0 +1,105 @@
+"""This example illustrates how to get started easily with the SimpleKGPipeline
+and ingest PDF into a Neo4j Knowledge Graph.
+
+This example assumes a Neo4j db is up and running. Update the credentials below
+if needed.
+
+It's assumed Ollama is used to run a model locally.
+"""
+
+import asyncio
+import ollama
+from pathlib import Path
+
+import neo4j
+from neo4j_graphrag.experimental.pipeline.kg_builder import SimpleKGPipeline
+from neo4j_graphrag.experimental.pipeline.pipeline import PipelineResult
+from neo4j_graphrag.llm import LLMInterface, LLMResponse
+
+from llama_index.embeddings.ollama import OllamaEmbedding
+from neo4j_graphrag.embeddings.base import Embedder
+
+
+class OllamaEmbedder(Embedder):
```

Comment:
- Why not using the OpenAIEmbeddings class, which is compatible with ollama? I think having this here might be misleading for the users, as they do not have to implement their own embedder to use Ollama.
- In general, does this Ollama stuff belong to this PR? I see you've also added the dependency to ollama?

Reply:
Ollama is only used here for manual testing. Removing now